pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "pyansys"
-version = "2023.2.dev0"
+version = "2024.1.dev0"
 description = "Pythonic interfaces to Ansys products"
 readme = "README.rst"
 requires-python = ">=3.7,<4"
```

Comment:
Should this impose Python>=3.8?

Reply:
Hmmm let's wait a little bit for that. Because dependabot will still ingest patch updates of the packages inside. And those patch packages are really for the 2023.2 release. Once we do the official release of 2023.2 then we change it =)